lib/miq_automation_engine/service_models/miq_ae_service_container_deployment_node.rb

```diff
@@ -0,0 +1,8 @@
+module MiqAeMethodService
+  class MiqAeServiceContainerDeploymentNode < MiqAeServiceModelBase
+    expose :vm_id
+    expose :is_tagged_with?
+    expose :tags
```

Comment:
Instead of exposing `is_tagged_with?` and `tags` use the tagging methods exposed to automate here:
https://github.com/ManageIQ/manageiq/blob/10c5b9d40809ca014c3c8b4f7f5d96903fb0597b/lib/miq_automation_engine/engine/miq_ae_service_model_base.rb#L232

The methods `tagged_with?` and `tags` are available on all service models.